member-service/src/test/java/com/hedvig/memberservice/external/trustpilot/TrustpilotOauth2InterceptorTest.kt

```diff
@@ -0,0 +1,112 @@
+package com.hedvig.memberservice.external.trustpilot
+
+import com.hedvig.memberservice.external.trustpilot.TrustpilotOauth2Interceptor.TokenResponse
+import feign.RequestTemplate
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.slot
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
```

Comment:
Personal opinion, when creating new tests I like using junit 5 and it looks like they will run if you would change this to and use `BeforeEach` annotation instead of `Before` :slightly_smiling_face:  But as mentioned Personal opinion
```suggestion
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test
```

Reply:
Aaah - I was used to `@BeforeEach` too - but I didn't get that to work. I realise now that it didn't because I was still using the Junit4 version of `@Test` of course